packages/devtools-reps/src/reps/tests/grip-array.js

```diff
@@ -57,18 +59,23 @@ describe("GripArray - max props", () => {
   it("renders as expected", () => {
     const renderRep = (props) => shallowRenderRep(object, props);
 
-    const defaultOutput = `Array [ 1, "foo", {} ]`;
+    let length = getGripLengthBubbleText(object);
```

Comment:
i'd rather not use the `getGripLengthBubbleText` function and hardcode the bubble in each test.
Here, if at some point `getGripLengthBubbleText` returns `""`, the test will pass although it should fail.

Reply:
@nchevobbe It may be kind of confusing at first sight, but to preemptively print the empty `""` length text makes the tests more future-proof. What if we change the `visibilityThreshold` prop for certain grips? What if we decide to change the `maxLengthMap` of `grip-array`/`grip-map` in the future? We'd need to update all tests by hand and omit or add `${length}` in certain expected outputs.

If we leave it as (calculate `${length}` for each mode of each stub regardless of empty text), we're having more dynamic test-cases, although not as straightforward as your proposal would make 'em. 🙂 

For reference, here are the current conditions that return `""`:
```javascript
  const isEmpty = length === 0;
  const isObvious = [MODE.SHORT, MODE.LONG].includes(mode) &&
    length <= maxLengthMap.get(mode) &&
    length <= visibilityThreshold;
  if (isEmpty || isObvious) {
    return "";
  }
```

Reply:
please tell me if I'm missing something! 😅

Reply:
I was saying return `""` as an example, if something changes in the function that might be unwanted.
But I see your point. So I think I will be okay with that if we add tests for `getGripLengthBubbleText`. That way, we can still use it in test, and if something goes wrong, the test will fail.